src/pages/sign-up-index-page/student-form.js

```diff
@@ -0,0 +1,51 @@
+const formRef = document.querySelector('form');
+const getInputRef = (name) => formRef.querySelector(`[name="${name}"]`);
+
+export const StudentForm = {
+  get formData() {
+    return new FormData(formRef);
+  },
+
+  get data() {
+    return {
+      name: this.formData.get('name'),
+      surname: this.formData.get('surname'),
+      birthday: this.formData.get('birthday'),
+    };
+  },
+
+  set data({ name, surname, birthday }) {
+    getInputRef('name').value = name;
+    getInputRef('surname').value = surname;
+    getInputRef('birthday').value = birthday;
+  },
+
+  get age() {
+    const birthDate = this.data.birthday;
+    const currentDate = new Date();
+    const [currentYear, currentMonth, currentDay] = [
+      currentDate.getFullYear(),
+      currentDate.getMonth(),
+      currentDate.getDate(),
+    ];
+    const [birthYear, birthMonth, birthDay] = birthDate.split('-').map((item) => +item);
```

Comment:
There might be risk at some moment for the format of this value to be changed, so I'd suggest using Date constructor instead and calling methods getFullYear etc.